### PR TITLE
Add retries to overcome Jenkins API flakiness

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit:
- Adds retries to the step where watcher creates/updates the jenkins
  jobs by hitting the API.
- Removes the old networking job in favor of uperf job.

Fixes https://github.com/openshift-scale/scale-ci-pipeline/issues/113

### CODE REVIEW

**Review by mohit-sheth — APPROVED** [review]
LGTM

File: scale-ci-watcher.sh [hunk 1]
```diff
@@ -117,8 +120,18 @@ function update_scale_ci_jobs() {
 			echo "--------------------------------------------------"
 			echo "INFO: Found $template"
 			echo -e "--------------------------------------------------\n"
-                	jenkins-jobs --conf "$JJB_CONFIG_PATH" update "$template"
-                	if [[ $? != 0 ]]; then
+			retries=0
+         		for (( iter=1; iter<=$watcher_retries; iter++ ))
```
Comment (dry923) [inline scale-ci-watcher.sh:124]: Not that I'm hoping for one, but does there need to be any type of wait/sleep interval between each attempt?

Comment (chaitanyaenr) [reply scale-ci-watcher.sh:125]: Makes sense, added a sleep interval between each attempt @dry923. PTAL.

**Review by dry923 — APPROVED** [review]
LGTM

**Pull request merged**